static/js/snap-pages/components/RegisteredSnaps/RegisteredSnaps.tsx

```diff
@@ -0,0 +1,175 @@
+import {
+  ConfirmationModal,
+  MainTable,
+  Notification,
+  Row,
+  Strip,
+  Tooltip,
+} from "@canonical/react-components";
+import { ISnap } from "../../types";
+import { useState } from "react";
+import { PAGE_NUMBER } from "../../types/constants";
+
+export const RegisteredSnaps = ({
```

Comment:
I think this should be a named function and the export at the bottom of the file to match other React components